File: week02/day03/diagonal-matrix.py
# - Create (dynamically) a two dimensional list
#   with the following matrix. Use a loop!
#
#   1 0 0 0
#   0 1 0 0
#   0 0 1 0
#   0 0 0 1
#
# - Print this two dimensional list to the output

x = 4
matrix = [[ 0 for i in range(x)] for j in range(x)]
for i in range(4):
    for j in range(4):
        if i == j:
            matrix[i][j] = 1
for i in matrix:
    print(i)

#matrix[0][0]=1

File: week07/day03/diagonal-matrix.js
'use strict';
// - Create (dynamically*) a two dimensional list
//   with the following matrix**. Use a loop!
//
//   0 0 0 1
//   0 0 1 0
//   0 1 0 0
//   1 0 0 0
//
// - Print this two dimensional list to the console
//
// * size should depend on a variable
// ** Relax, a matrix is just like an array

var matrix = []

for (var i = 0; i < 4; i++) {
  matrix[i] = [];
  for (var j = 0; j<4; j++) {
    matrix[i][j] = 0;
    if ( i === j) {
      matrix[i][j] = 1
    }
  }
}


console.log(matrix);
